howso/direct/core.py

```diff
@@ -1649,6 +1650,15 @@ def batch_react_series(
             maximum size a series is allowed to be.  Default is
             3 * model_size, a 0 or less is no limit. Must provide
             either one for all series, or exactly one per series.
+        continue_series : bool, default False
+            When true will attempt to continue existing series instead of
+            starting new series. If initial_values provide series IDs, it
+            will continue those explicitly specified IDs, otherwise it will
+            randomly select series to continue.
+            .. note::
+
+                Terminated series with terminators cannot be continued and
+                will result in null output.
         derived_context_features : iterable of str, optional
             list of context features whose values should be computed
```

Comment:
I know you didn't do this, if you don't mind, can you capitalize List and rest of the uncapitalized parameter definitions in that section.